client/src/components/DataFiles/DataFilesProjectFileListing/DataFilesProjectFileListing.module.scss

```diff
@@ -42,3 +49,39 @@
   padding-left: 0.25em;
   padding-right: 0.25em;
 }
+
+@media only screen and (max-width: 2399px) {
+  .title {
+    max-width: 1150px;
+  }
+}
+
+@media only screen and (max-width: 1919px) {
+  .title {
+    max-width: 890px;
+  }
+}
+
+@media only screen and (max-width: 1779px) {
+  .title {
+    max-width: 770px;
+  }
+}
+
+@media only screen and (max-width: 1679px) {
+  .title {
+    max-width: 470px;
+  }
+}
+
+@media only screen and (max-width: 1199px) {
+  .title {
+    max-width: 270px;
+  }
+}
+
+@media only screen and (max-width: 991px) {
+  .title {
+    max-width: 200px;
+  }
+}
```

Comment:
__Breakpoints__

Please prefer the [Shared UI - Constants - Breakpoints](https://confluence.tacc.utexas.edu/x/b4AZCg) for media query values. Example usage: [`/client/src/components/Jobs/Jobs.scss`](https://github.com/TACC/Core-Portal/blob/main/client/src/components/Jobs/Jobs.scss). If usage of these suggested values is not suitable, let me know.

<details>
<summary>Why?</summary>

Consistent media queries allow responsive design to occur at predictable screen widths, which can ease design decisions for dev and design.

Eventually, Portal will use [re-usable media queries (like CMS)](https://github.com/TACC/Core-CMS/blob/main/taccsite_cms/static/site_cms/css/src/_imports/tools/media-queries.css) (so we need not remember these values).

</details>